nebula-logger/tests/log-management/classes/LoggerSettingsController_Tests.cls

```diff
@@ -0,0 +1,378 @@
+        Integer expectedLoggingLevelSize = LoggingLevel.values().size() - 1; // LoggingLEVEL.NONE and LoggingLevel.INTERNAL are ignored, '--NONE--' is automatically included
+        List<LoggerSettingsController.PicklistOption> picklistOptions = LoggerSettingsController.getPicklistOptions().loggingLevelOptions;
+        System.assertEquals(expectedLoggingLevelSize, picklistOptions.size());
+        for (LoggerSettingsController.PicklistOption picklistOption : picklistOptions) {
+            if (String.isBlank(picklistOption.value) == true) {
+                System.assertEquals('--None--', picklistOption.label);
+            } else {
+                System.assertEquals(picklistOption.value, picklistOption.label);
+                LoggingLevel matchingLoggingLevel = LoggingLevel.valueOf(picklistOption.value);
+                System.assertEquals(matchingLoggingLevel.name(), picklistOption.label);
+            }
+        }
+    }
+
+    @IsTest
+    static void it_should_return_loggerSaveMethod_picklist_options() {
+        Integer expectedLoggerSaveMethodSize = Logger.SaveMethod.values().size() + 1; // '--NONE--' is automatically included
+        List<LoggerSettingsController.PicklistOption> picklistOptions = LoggerSettingsController.getPicklistOptions().saveMethodOptions;
+        System.assertEquals(expectedLoggerSaveMethodSize, picklistOptions.size());
+        for (LoggerSettingsController.PicklistOption picklistOption : picklistOptions) {
+            if (String.isBlank(picklistOption.value) == true) {
+                System.assertEquals('--None--', picklistOption.label);
+            } else {
+                System.assertEquals(picklistOption.value, picklistOption.label);
+                Logger.SaveMethod matchingLoggerSaveMethod = Logger.SaveMethod.valueOf(picklistOption.value);
+                System.assertEquals(matchingLoggerSaveMethod.name(), picklistOption.label);
+            }
+        }
+    }
+
+    @IsTest
+    static void it_should_return_setupOwnerType_picklist_options() {
+        List<String> expectedSetupOwnerType = new List<String>{ 'Organization', 'Profile', 'User' };
+        Integer expectedSetupOwnerTypeSize = 4; // Options are org, profile, and user, and '--NONE--' is automatically included
+        List<LoggerSettingsController.PicklistOption> picklistOptions = LoggerSettingsController.getPicklistOptions().setupOwnerTypeOptions;
+        System.assertEquals(expectedSetupOwnerTypeSize, picklistOptions.size());
+        for (LoggerSettingsController.PicklistOption picklistOption : picklistOptions) {
+            if (String.isBlank(picklistOption.value) == true) {
+                System.assertEquals('--None--', picklistOption.label);
+            } else {
+                System.assertEquals(picklistOption.value, picklistOption.label);
+                System.assertEquals(true, new Set<String>(expectedSetupOwnerType).contains(picklistOption.value));
```

Comment:
any reason you're converting to a set here? if this was in production-level code and it was a big list, I wouldn't use `List.contains()` but for a test with only a few strings in it, I think it's fine. (same with the below test(s))

Reply:
I think these lists are all small enough that I'm not too worried about using `.contains()`.... but I'm not sure why I defined the variables as `List<>` and then just convert to `Set<>`, that part seems silly. I'll change the variables to be initially defined as `Set<>`